indico/modules/events/reminders/templates/preview_text.html

```diff
@@ -2,4 +2,4 @@
 <pre class="mono">{{ subject }}</pre>
 <br>
 <strong>{% trans %}Body{% endtrans %}</strong>
-<pre class="mono">{{ body }}</pre>
+<pre class="mono">{{ body | safe }}</pre>
```

Comment:
That sounds like a great way to self-XSS. You most likely want the `sanitize_html` filter instead.

Reply:
Also, why does the text/plain preview need to render any HTML tags? I wouldn't expect this to be necessary...